attribute-ui/src/components/App.tsx

```diff
@@ -24,8 +26,27 @@ export const App = () => {
     const serverUrl = window.SERVER_URL;
     const hostToken = window.location.pathname.substr(1);
 
-    const poll = (): Promise<SessionAttributes> => fetch(`${serverUrl}/session_info/${hostToken}`).then(r => r.json())
+    const poll = (): Promise<SessionAttributes> => fetch(`${serverUrl}/session_info/${hostToken}`, {
+        credentials: 'include',
+    }).then(r => {
+        if (r.status == 401) {
+            r.json().then(j => {
+                setLoginUrl(j.detail + '');
+            });
+            return [];
```

Comment:
It seems illogical to not mark us as unauthorized when receiving a 401, since we are at that point unauthorized, just with a way to get out of that.

Reply:
Changed to use `Status.Unauthenticated` and `Status.Unauthorized`